nmostesting/mocks/Node.py

```diff
@@ -211,6 +212,16 @@ def parse_sdp(self, sdp_data):
                 sdp_param_leg["source_ip"] = filter_line.group(2)
             else:
                 sdp_param_leg["source_ip"] = None
+
+            format_line = re.search(r"a=fmtp:(\S*\s*)(.*)", sdp_data)
+
+            if format_line and format_line.group(2):
+                #  Handle parameter keys that have no value, e.g. 'interlace' and set the value to 'True'
+                sdp_param_leg["format"] = {key: ''.join(value) if len(value) > 0 else 'True'
```

Comment:
`''.join(value)` will concatenate the elements of the list directly whereas there were produced using `item.split('=')`, so although it's unlikely there were `=` characters in the value in the first place, I think you want `'='.join(value)`... or, probably better, to just use `item.split('=', maxsplit=1)` and then use `value[0]` here?

Reply:
Hmmm, yeah the intent here is to be able to handle parameters that are not key, value pairs (separated by an '='), namely `interlace`.  Admittedly the expression is slightly impenetrable - it interprets each split of the format string as an array which will have one or zero elements. The `''.join` value then converts the single element array to a value, and the empty array fails the test and is set to `True`.  There must be a more elegant way of handling this though.....I'll give it some thought

Reply:
Like I said, you've assumed zero or one but unless you use `maxsplit=1` that risks making an ass out of u and me 😉

Reply:
Now changed.